packages/x-tailwindcss/demo/options.ts

```diff
@@ -61,12 +62,7 @@ export const options: PluginOptions = {
     return {};
   },
   theme: {
-    colors: {
-      neutral: {
-        10: 'red',
-        20: 'orange'
-      }
-    },
+    colors: {},
```

Comment:
@javieri-empathy if you can take a look into this types. If you add:
```js
   '.button': {
      '&.outlined': {
        '&.primary': {}
      }
    }
```
to this options, `.button` is autocomplete, `&.outlined`, is autocomplete but `&.primary` not, event the function `mapColors` has the right type.

I think it is because the `DeepPartial` in the type `ComponentsDefinition`, but I'm not sure.